tables/alb_access_log/alb_access_log.go

```diff
@@ -0,0 +1,230 @@
+package alb_access_log
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type AlbAccessLog struct {
+	schema.CommonFields
+
+	ActionsExecuted        string    `json:"actions_executed,omitempty"`
```

Comment:
Should this be a list of strings instead, since the original log data is either `-` or a comma-separated list of actions taken?